addons/sys_core/initSettings.sqf

```diff
@@ -11,17 +11,27 @@
 [
     QGVAR(premixGlobalVolume),
     "SLIDER",
-    "Pre-Mix Global Volume",
+    localize LSTRING(premixGlobalVolume_displayName),
     "ACRE2",
     [0, 3, 1, 2],
     false,
     {["premixGlobalVolume", _this] call FUNC(setPluginSetting)}
 ] call CBA_Settings_fnc_init;
 
 [
+    "ACRE_SPECTATOR_VOLUME",
+    "SLIDER",
+    localize LSTRING(ACRE_SPECTATOR_VOLUME_displayName),
+    "ACRE2",
+    [0, 1, 1, 2],
+    false,
+    {["premixGlobalVolume", _this] call FUNC(setPluginSetting)}
```

Comment:
In this case the fix should be just replacing this line with `{}`. The function `FUNC(setPluginSetting)` is for sending values to the teamspeak plugin for settings relevant to the plugin. But in the case of `ACRE_SPECTATOR_VOLUME` it only needs setting in SQF land and will also have almost immediate effect on spectator volume 👍